app/controllers/api/v1/simulator_controller.rb

```diff
@@ -20,6 +20,7 @@ def post_issue
 
     response = HTTP.post(url, json: { text: text })
```

Comment:
Hi @Recxsmacx, you need to configure sentry for the simulator javascript client side code, [reference](https://github.com/CircuitVerse/CircuitVerse/tree/master/simulator/src).